### ISSUE TITLE
add export of util.js

### ISSUE THREAD

**Issue by koola** [fent/node-ytdl-core#97]
Would be nice to have this to avoid the extra require

```javascript
'use strict';

const util = require('./node_modules/ytdl-core/lib/util.js'),
    ytdl = require('ytdl-core');

ytdl.getInfo('https://www.youtube.com/watch?v=OpY53_agpDk', (err, info) => {
    if (err) {
        throw err;
    } else {
        console.log(util.chooseFormat(info.formats, 
            { filter: fmt => { return fmt.container === 'mp4'; }}));
    }
});
```

**Comment by fent** [comment]
I didn't realize people were using these util functions this way. I'll add this soon.

**Comment by fent** [comment]
Published v0.9.0
